.github/workflows/run_recipes.yml

```diff
@@ -35,7 +35,8 @@ jobs:
       - name: Run recipes
         run: |
           python Simulations/python/downloadPackages.py
-          python Simulations/python/run_recipes.py --small --doCalib=1 --sequence=1
+          cd Simulations
```

Comment:
This line, `cd Simulations` should go up one line, because now the script first downloads the METIS package, and then enters a different directory where the package are not.

What it should do: first enter the directory, then download the packages.

I cannot make a suggestion on github here, because the downloadPackages line has not been changed.

Reply:
See #119